docs/faq/README.md

```diff
@@ -138,10 +171,22 @@ If you are using Enterprise on-premises, please write an email to sales @ openpr
 
 If you are using the Enterprise cloud, you can easily upgrade or downgrade the number of users by navigating to *Administration -> Billing -> Manage subscription* and choosing the new amount of users which you need in your system. Find out more [here](../enterprise-guide/enterprise-cloud-guide/manage-cloud-subscription).
 
+### How can I check the workload of each of my employees?
+
+The best way is to use one of our Action boards, the [Assignee board](.../.../.../agile-boards/#choose-between-board-types). Alternatively, you can filter work package lists (in each project or using the global work package list accessible via Modules in the right part of the header) to filter for work packages where employees are set as Assignee or as Accountable.
+
 ### How can I change the day my week starts with, etc.?
 
 You can do this as a system administrator in the [System settings](../system-admin-guide/system-settings/display-settings/#time-and-date-formatting-aggregation-of-changes-in-activity).
 
+### How to use plugins in OpenProject and where can I find them?
+
+Not all available plugins for OpenProject are published yet. After updating the plugins so that they are working with the Rails3 OpenProject Core, they will be published on Github continiously. Please write us a note if you have questions to any of the plugins.
+
+### Which Plugins are published so far?
+
+Please refer to the project timeline to see which plugins will be released. Plugin development will take place in dedicated sub-projects on openproject.org.
+
```

Comment:
Please refer to the list of [integrations and Community plugins](../system-admin-guide/integrations/)